src/input.cpp

```diff
@@ -116,15 +145,15 @@ namespace openre::input
         joy_get_pos_ex(gGameTable.input.mapping);
         if (gGameTable.input.var_1F8 != 0)
         {
-            v1 = sub_43BAC0(gGameTable.input.var_24, 0);
+            v1 = get_input_device_state(gGameTable.input.var_24, INPUT_DEVICE_KEYBOARD);
 
             gGameTable.dword_99CF64 = gGameTable.input.var_24;
             gGameTable.dword_66D394 = v1;
         }
         gGameTable.dword_99CF70 = 0;
         if (gGameTable.input.var_3B24 >= 2 && gGameTable.input.var_3D0 != 0)
         {
-            auto v2 = sub_43BAC0(gGameTable.input.var_1FC, 1);
+            auto v2 = get_input_device_state(gGameTable.input.var_1FC, INPUT_DEVICE_CONTROLLER);

```

Comment:
var_24 could be renamed to keyboard_raw_state.
var_1FC could be renamed to gamepad_raw_state.